### DISCUSSION TITLE
Add element 'masking' to current capabilities

### DISCUSSION THREAD

**Discussion by unlikelyzero** [percy/cli#1030] [Ideas] [answered]
## Enhancement
We're currently considering moving some of our percy tests to playwright's built in snapshot capabilities due to the 'mask' capabilities which were recently added
https://playwright.dev/docs/test-assertions#page-assertions-to-have-screenshot-1-option-mask

I've used Applitools in the past and the eyes.check().ignore() property was something we leaned on pretty heavily https://applitools.com/docs/topics/sdk/the-eyes-sdk-check-fluent-api.html?Highlight=ignore

Are there any plans to add an ignore or mask property to the sdk?

**Comment by djones** [comment, marked as answer]
Hey @unlikelyzero 👋🏻, thanks for the suggestion.

> Are there any plans to add an ignore or mask property to the sdk?

Sure are! We're currently working on a feature along these lines. No promises on when it'll be ready, but stay tuned.
